tools/deployment/osqueryctl

```diff
@@ -77,7 +81,8 @@ start() {
 
 stop() {
   if [ $OS = "darwin" ]; then
-    launchctl stop $PLIST_PATH
+    launchctl unload $PLIST_PATH
+    rm $PLIST_PATH
```

Comment:
I was a little confused about how you were supposed to use `osqueryctl start` and `osqueryctl stop`.

The package building script doesn't actually drop the plist in `/Library/LaunchDaemons`,  (it goes so `/var/osquery`), so it was necessary to copy it over first. Also (looking at `man launchctl`), it looks like the start command is just supposed to be used for debugging. Is there a reason why you used `start` and `stop` instead of `load` and `unload`?